django_material_demo/templates/material/frontend/views/filter_list.html

```diff
@@ -5,7 +5,7 @@
 <div class="right-panel">
     <div class="card">
     <dmc-form>
-    <form method="GET">
+    <form method="GET" action="./">
```

Comment:
Isn't the default action current path? I don't think it's needed.

Reply:
This is added to prevent accumulating GET params caused by multiple form submissions